charts/team-ns/templates/builds/buildpack.yaml

```diff
@@ -67,7 +67,11 @@ spec:
       value: {{ . }}
     {{- end }}
     - name: BUILDER_IMAGE
-      value: paketobuildpacks/builder:full
+      value: paketobuildpacks/builder-jammy-full:latest
```

Comment:
Is there an explicit version for paketobuildpacks/builder-jammy-full instead of latest?

Reply:
Got it. I am fine with it.
If we go for the latest tag then most probably we need to create an exception for policy check.
The Pods that perform build jobs should have `policy.otomi.io/ignore: banned-image-tags` annotation.